.idea/vcs.xml

```diff
@@ -1,24 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!--
```

Comment:
unnecessary change?

Reply:
btw @shardulm94 you can do this to tell `git` to ignore local changes to this file:
```
git update-index --skip-worktree .idea/vcs.xml
```

Reply:
My bad! IntelliJ modified the file and I didn't realize when pushing an empty commit. @xkrogen Thanks for the tip!